src/AWS.Deploy.CLI/AWS.Deploy.CLI.csproj

```diff
@@ -43,6 +43,4 @@
     <None Include="..\..\icon.png" Pack="true" PackagePath="" />
   </ItemGroup>
 
-  <Import Project="..\AWS.Deploy.Constants\AWS.Deploy.Constants.projitems" Label="Shared" />
```

Comment:
Why the change here and in the sln? I'm unsure how it's related.

Reply:
This is in response to Norm's comment about using the orchestrator to get to cdkprojecthandler. cdkprojecthandler is marked internal in the orchestrator, so i had to mark its internals visible to the CLI. By doing that I cause a conflict related to the constants project, so I had to remove it's reference from the CLI project as it can access them through the orchestrator.